### PR TITLE
Release v1.6.0

### PR DESCRIPTION
**Pull request by Jack-Works**
# Fresh install tests
- [x] On Firefox, as a new user
- [x] On Chrome, as a new user
- [x] On Firefox, as a return user
- [x] On Chrome, as a return user

# Upgrade install tests

- First, install and setup by release 1.5.3, then
- [x] On Chrome, upgrade to 1.6.0 ( @Jack-Works )
- No need to test Firefox (No previous formal release on Firefox)

# Test on Mobile
- No need to test since there is no formal release for the mobile version.
- [ ] Optional: Firefox for Android

# Release
- [ ] changelog
    - [ ] Release on Github
- [ ] Release new version on Chrome Web Store
- [ ] Release on Firefox Add-ons ( #25 )

### CODE REVIEW

**Comment by Jack-Works** [comment]
TBD
- [ ] Hide incomplete twitter support
- [ ] Fix auto paste into post box

**Comment by SunriseFox** [comment]
+ Yarn watch

- [ ] Lint error detach processes from console, but not quit the server daemon.
- [ ] Lint requires more than 25s to finish on a i7-8750H model.

+ Chrome, new user: 

- [ ] after creating persona, choose **Add public key to profile**, `m.facebook.com` opened instead of desktop verson.
- [ ] after creating persona, choose **Create a verification post**, key is pasted to the first post's comment box instead of the post box.
- [ ] open popup got  white page, `Target container is not a DOM element` at unknown source.
- [ ] `Developer` tab on the options page is confusing
- [ ] magic decrypt result:

   image

**Comment by Jack-Works** [comment]
To be done before the release:

- [ ] Check automatically find prove post/bio on Chrome Desktop
- [ ] Check automatically find prove post/bio on Firefox
- [ ] Check automatically find prove post/bio on Firefox Mobile

- [ ] Add two-way key/bio search from gun in UI

**Comment by SunriseFox** [comment]
- [ ] automatically finding prove post/bio opens a not-found page
- [ ] cannot decrypt any post when first seeing it
- [ ] when trying to decrypt, got `decryptMessage1ToNByOther cannot resolve an available key`

**Comment by SunriseFox** [comment]
- [ ] On firefox, Maskbook prevents facebook page from navigating and infinite scrolling, then stops page from responding, and eventually firefox prompts to stop it.
image

- [ ] PostId may contains '/' but we are currently rejecting it.

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -51,6 +51,12 @@
     "decrypted_postbox_decrypting": {
         "message": "Maskbook 解密中…"
     },
+    "decrypted_postbox_decrypting_finding_person_key": {
+        "message": "Maskbook 正在搜尋對方的公鈅..."
```
Comment (Artoria2e5) [inline src/_locales/zh/messages.json]: `…`

**Comment by Jack-Works** [comment]
Note: don't forget to add `tabs` permission on Firefox

**Pull request merged**